tests/lang/s02_core/s01_procedures/t01_regular_procs.nim

```diff
@@ -87,6 +92,13 @@ block forward_declare:
   doAssert forward() == 12
   doAssert callsForward() == 12
 
+block invalid_forward_declare:
```

Comment:
This needs to be in a separate failing test?

Reply:
I'll remove it fn xoxo